test/index.js

```diff
@@ -3,6 +3,7 @@
 var chai = require('chai')
 var expect = chai.expect
 var MarkdownIt = require('markdown-it')
+var Imsize = require('@centerforopenscience/markdown-it-imsize')
```

Comment:
Rather than adding a new dev dependency, let's just create a simple inline plugin, especially since I'd rather make the solution more generic than specific to a particular plugin.